mappings/net/minecraft/entity/vehicle/AbstractMinecartEntity.mapping

```diff
@@ -1,9 +1,18 @@
 CLASS net/minecraft/class_1688 net/minecraft/entity/vehicle/AbstractMinecartEntity
 	CLASS class_1689 Type
+	FIELD field_7655 yVelocityClient D
+	FIELD field_7656 zVelocityClient D
+	FIELD field_7657 pitchClient D
+	FIELD field_7658 xVelocityClient D
+	FIELD field_7659 yawClient D
 	FIELD field_7661 CUSTOM_BLOCK_OFFSET Lnet/minecraft/class_2940;
+	FIELD field_7662 zPosClient D
 	FIELD field_7663 DAMAGE_WOBBLE_TICKS Lnet/minecraft/class_2940;
+	FIELD field_7665 xPosClient D
+	FIELD field_7666 yPosClient D
```

Comment:
These should probably reflect the regular position fields and drop the `pos` part. (xClient)

Reply:
Yeah I actually disagree with these minecart field mappings as well